dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/services/DatabaseStringDataStore.kt

```diff
@@ -176,6 +178,18 @@ class DatabaseStringDataStore(
         logger.info("Inserting data into database with data ID: $dataId and correlation ID: $correlationId.")
         storeDataItemWithoutTransaction(DataItem(dataId, objectMapper.writeValueAsString(data)))
         publishStorageEvent(payload, correlationId)
+
+        val bypassQa = JSONObject(payload).getBoolean("bypassQa")
+        val body =
+            objectMapper.writeValueAsString(
+                ManualQaRequestedMessage(
+                    resourceId = dataId,
+                    bypassQa = bypassQa,
+                ),
+            )
+        cloudEventMessageHandler.buildCEMessageAndSendToQueue(
+            body, MessageType.MANUAL_QA_REQUESTED, correlationId, ExchangeName.ITEM_STORED, RoutingKeyNames.DATA_QA,
+        )
```

Comment:
I thought the `Manual QA Requested` message was removed when the auto QA service got deprecated. Doesn't the QA service directly listen on the Data received messages?

Reply:
I will contact you directly